src/bp/core/routers/admin/bots.ts

```diff
@@ -70,6 +77,20 @@ export class BotsRouter extends CustomRouter {
       })
     )
 
+    router.get(
+      '/byWorkspaces',
+      assertSuperAdmin,
```

Comment:
instead of asserting that the user is superAdmin, we you could scope down the workspaces to which the user has access ? 

Imagine someone that is an admin of 2 or 3 different workspaces but isn't the superhero of the whole cluster

Reply:
That method is only used in the monitoring page, which requires the user to be super admin. The only purpose is to know what workspace a bot is in